src/main/webapp/metadataFragment.xhtml

```diff
@@ -64,12 +64,10 @@
                                         </p>
                                     </ui:fragment>
                                     <ui:repeat value="#{dsf.datasetFieldCompoundValues}" var="compoundValue">
-                                        <ui:repeat value="#{compoundValue.displayValueMap.entrySet().toArray()}" var="cvPart" varStatus="partStatus">
+                                        <ui:repeat value="#{compoundValue.displayValueMap.entrySet().toArray()}" var="cvPart" varStatus="partStatus">                                           
                                             <h:outputText value="#{dsf.datasetFieldType.displayFormat} " rendered="${!partStatus.first}"/>
```

Comment:
When applied to a compound field, the display format is being shown in as escaped.

![darwin_s_finches_-_dvc565817e](https://user-images.githubusercontent.com/1036060/27877474-c3a62e6c-6188-11e7-8796-bd202b1f42d5.png)